mk/mcu/STM32H5.mk

```diff
@@ -0,0 +1,227 @@
+#
+# H5 Make file include
+#
+
+ifeq ($(DEBUG_HARDFAULTS),H5)
+CFLAGS          += -DDEBUG_HARDFAULTS
+endif
+
+#CMSIS
+CMSIS_DIR      := $(ROOT)/lib/main/CMSIS
+
+#STDPERIPH
+STDPERIPH_DIR   = $(ROOT)/lib/main/STM32H5/Drivers/STM32H5xx_HAL_Driver
+STDPERIPH_SRC   = $(notdir $(wildcard $(STDPERIPH_DIR)/Src/*.c))
+
+EXCLUDES        = \
```

Comment:
Isn't it better to white-list driver files? It will make git history cleaner. And make unwanted file inclusion harder.

Reply:
Agree. I'll update this PR, and adjust the other MK files in another PR.